### PR TITLE
PAS-421 | Add community SDKs

### PR DESCRIPTION
**Pull request by Tyrrrz**
https://bitwarden.atlassian.net/browse/PAS-421

### CODE REVIEW

File: src/guide/backend/index.md [hunk 1]
```diff
@@ -8,3 +8,9 @@ You can use Passwordless.dev with any programming language by implementing calls
 - [PHP](php.md)
 - [Python 2](python2.md)
 - [Python 3](python3.md)
+
+Unofficial community SDKs:
+
+- [Rust (by davidzr)](https://github.com/davidzr/passwordless-rust)
```
Comment (jonashendrickx) [inline src/guide/backend/index.md:14]: @Tyrrrz Should we add something related to our responsibility with regards to security and correct functioning of the SDKs?

Comment (Tyrrrz) [reply src/guide/backend/index.md:14]: I think it's kind of implied in the word "unofficial". What do you suggest?

File: src/guide/backend/index.md [hunk 2]
```diff
@@ -8,3 +8,9 @@ You can use Passwordless.dev with any programming language by implementing calls
 - [PHP](php.md)
 - [Python 2](python2.md)
 - [Python 3](python3.md)
+
+Unofficial community SDKs:
+
```
Comment (abergs) [inline src/guide/backend/index.md:13]:
```suggestion

These SDK's are developed by the community. They are not developed by, supported or audited by Bitwarden.

```

File: src/guide/backend/index.md [hunk 3]
```diff
@@ -8,3 +8,11 @@ You can use Passwordless.dev with any programming language by implementing calls
...
+These SDKs are developed by the community. They are not developed by, supported or audited by Bitwarden.
```
Comment (abergs) [inline src/guide/backend/index.md:14]:
```suggestion
These SDKs are developed by the community. They are not developed, supported or audited by Bitwarden.
```

**Review by abergs — APPROVED** [review]
I added a small paragraph. Approved.

**Pull request merged**